### PR TITLE
Add a class specificity for SplFileInfo text

### PR DESCRIPTION
**Pull request by rendler-denis**
- update text to differentiate between PHP's SplFileInfo and Symfony's SplFileInfo

### CODE REVIEW

File: components/finder.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ directories::
     }
 
 The ``$file`` is an instance of :class:`Symfony\\Component\\Finder\\SplFileInfo`
-which extends :phpclass:`SplFileInfo` to provide methods to work with relative
+which extends PHP's own :phpclass:`\SplFileInfo` to provide methods to work with relative
```
Comment (wouterj) [inline components/finder.rst]: the `\` should be removed

Comment (rendler-denis) [inline components/finder.rst]: Issue resolved :) 
I added it just to provide a FQN context.

**Comment by wouterj** [comment]
👍 

status: reviewed

**Comment by weaverryan** [comment]
I like it! Thanks Denis!

**Pull request closed without merging**